script.js

```diff
@@ -1,7 +1,19 @@
 // PopUp Object
 
 const popObjects = [
-
+  {
+    title: 'Awesome books',
+    subtitle: 'Microverse',
```

Comment:
- [ ] @asadmisrani, I think the same issue is still persisting, you should see how to fix it on the go, see the record below and make the necessary changes to it

![6](https://user-images.githubusercontent.com/77843050/231710408-5b2f4838-d72a-494c-a049-46143f2f224d.gif)
- Seeing this record, the image only displays on the 1st card, the rest does not show up.